tools/mo/requirements.txt

```diff
@@ -1,6 +1,6 @@
 numpy>=1.16.6
 networkx~=2.5; python_version <= "3.6"
-networkx<=2.8.8; python_version > "3.6"
+networkx<3.1.0; python_version > "3.6"
```

Comment:
Propose to optimize `networks` dependencies by removing `python_version > "3.6"` and removing `networkx~=2.5; python_version <= "3.6"`
```suggestion
networkx<=2.8.8
```

Reply:
may be we can consider `<3.0.0`, we need to test it

Reply:
here is release notes https://networkx.org/documentation/stable/release/release_3.0.html#api-changes

Based on this we can make decisions whether MO is affected or we try to minimal effort migrate to 3.x release series.

Reply:
Let me take a look, thanks

Reply:
looks like the tests have passed

Reply:
what version of `networkx` is used? A bit risky to uodate higher `3.0` if it is not tested on such version

Reply:
We have a vicious circle:
- we cannot merge before validation
- we cannot validate before merge

Do we have MO tests in precommit?